src/frontend/js/components/SaleTunnel/AddressSelector/index.tsx

```diff
@@ -36,6 +36,14 @@ export const AddressSelector = () => {
       addresses.items.map((address) => ({
         value: address.id,
         label: getAddressLabel(address),
+        render: () => (
+          <div className="address-selector__option">
+            {address.title && (
+              <div className="address-selector__option__title fw-bold">{address.title}</div>
+            )}
+            <div>{getAddressString(address)}</div>
+          </div>
+        ),
```

Comment:
Let's improve semantic. Furthermore, I'll display address detail smaller than the title.

<img width="418" alt="image" src="https://github.com/openfun/richie/assets/9265241/f515a233-55bf-4c36-913b-6f448e13e2c9">

```suggestion
          <address className="address-selector__option">
            {address.title && (
              <strong className="address-selector__option__title">{address.title}</strong>
            )}
            <br />
            <small>{getAddressString(address)}</small>
          </address>
        ),
```